mlrun/datastore/targets.py

```diff
@@ -1069,6 +1064,47 @@ def prepare_spark_df(self, df, key_columns, timestamp_key=None, spark_options=No
         return df
 
 
+class ParquetTargetStoreyWrapper(storey.ParquetTarget):
+    def __init__(
+        self,
+        path: str,
+        index_cols: Union[str, Union[list[str], list[tuple[str, str]]], None] = None,
+        columns: Union[str, Union[list[str], list[tuple[str, str]]], None] = None,
+        partition_cols: Union[
+            str, Union[list[str], list[tuple[str, int]]], None
+        ] = None,
+        time_field: Union[None, str, int] = None,
+        time_format: Optional[str] = None,
+        infer_columns_from_data: Optional[bool] = None,
+        max_events: Optional[int] = None,
+        flush_after_seconds: Union[int, float, None] = None,
```

Comment:
Probably better to just forward `*args` and `**kwargs`then repeating all the parameters here.